lib/datadog/tracing/distributed/baggage.rb

```diff
@@ -125,6 +130,33 @@ def parse_baggage_header(baggage_header)
           end
           baggage
         end
+
+        # Convert baggage items to trace_distributed_tags with 'baggage.' prefix
+        # based on the baggage_tag_keys configuration
```

Comment:
Can you change this documentation so that it tells us the "why" for the reason we are generating span tags from baggage values?